src/main/blackbox/blackbox_io.c

```diff
@@ -59,6 +59,10 @@
 #include "drivers/sdcard.h"
 #endif
 
+#ifdef USE_BLACKBOX_VIRTUAL
+#include "blackbox_virtual.h"
+#endif
```

Comment:
I don't think this macro is needed here. If there is no use of `blackbox_virtual.h` in this file, the compiler will just ignore it.
```suggestion
#include "blackbox_virtual.h"
```